galaxy/reports.yml.sample

```diff
@@ -11,10 +11,10 @@ reports:
   #database_connection: postgresql://galaxy:galaxy@localhost:5432/galaxy?client_encoding=utf8
 
   # Where dataset files are stored.
-  #file_path: database/files
+  file_path: database/files
 
   # Where temporary files are stored.
-  #new_file_path: database/tmp
+  new_file_path: database/files
```

Comment:
Why this?

Reply:
I thought of keep it consistent with galaxy [config](https://github.com/bgruening/docker-galaxy-stable/blob/42e298e95e3c1064b5f13e9467a0f388aed569c5/galaxy/Dockerfile#L247). I will revert it back.

Reply:
Upstream is setting it to https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/galaxy.yml.sample#L495  so I guess we should change the ENV var instead.